test/unit/spec/MfaVerify_spec.js

```diff
@@ -214,8 +212,23 @@ function (Okta,
       return setup(res);
     }
 
+    var getAutoPushResponse = function (response, factorType, provider, autoPushVal) {
```

Comment:
same comment as in the other spec, it seems to me we don't need the 2nd and 3rd params.
```
var getAutoPushResponse = function (response, autoPushVal) {
```